pypdf/_writer.py

```diff
@@ -384,7 +384,7 @@ def viewer_preferences(self) -> Optional[ViewerPreferences]:
                 self._root_object[NameObject(CD.VIEWER_PREFERENCES)] = o
         return o
 
-    def create_viewer_preference(self) -> ViewerPreferences:
+    def create_viewer_preferences(self) -> ViewerPreferences:
```

Comment:
This is potentially a breaking change. as it's only been there for two weeks and not a feature I expect to be immediately adapted, I'll not make this a major version bump. Still something to point out in the commit message / top of PR.

Reply:
I've just noticed that it was in the title of the PR :+1:

Reply:
Thanks. Yeah, I had hoped this would make it in before an actual release. Seems better to me to deal with this now than have an inconsistent name in the API forever.